.github/workflows/linux.yaml

```
@@ -58,4 +52,4 @@ jobs:
           docker-compose -f spec/docker-compose.yml up -d
           bundle exec r10k puppetfile install
       - name: Run tests with expensive containers
```

```suggestion
      - name: Run integration tests
```

We can simplify/clarify these step names.